spec/services/teacher_training_public_api/sync_sites_spec.rb

```diff
@@ -120,22 +45,22 @@
     end
 
     context 'when the site exists' do
-      let!(:existing_site) { create(:site, provider:, uuid:, code: 'Old') }
+      let!(:site) { create(:site, provider:, uuid:, code: 'Old') }
 
       it 'does not create a new record' do
         expect { perform_job }.not_to change(Site, :count)
       end
 
       it 'updates the site in the db' do
         perform_job
-        site = Site.find_by(uuid:)
-        expect(site).to eq existing_site
-        expect(site.code).to eq site_code
+        found_site = Site.find_by(uuid:)
+        expect(found_site).to eq site
+        expect(found_site.code).to eq site_code
       end
 
       context 'course options already exist' do
-        let!(:course_option_1) { create(:course_option, site: existing_site, course:, study_mode: 'full_time') }
-        let!(:course_option_2) { create(:course_option, site: existing_site, course:, study_mode: 'part_time') }
+        let!(:course_option_1) { create(:course_option, :full_time, site:, course:) }
+        let!(:course_option_2) { create(:course_option, :part_time, site:, course:) }
 
         it 'updates existing course options' do
```

Comment:
I know you didn't change this but is this test description wrong or is it expecting the wrong thing? 🤔